pkg/agent/openflow/pipeline.go

```diff
@@ -2581,6 +2581,23 @@ func pipelineClassifyFlow(cookieID uint64, protocol binding.Protocol, pipeline b
 		Done()
 }
 
+// igmpEgressFlow generates flows to match IGMP report to jump to table MulticastRoutingTable.
+// This is because normal multicast egress rule can match IGMP v1 report, when there is egress
+// rule to block multicast traffic, IGMP v1 report will also be blocked, which is not expected.
+// Another issue, is that multicast stats will also count IGMP v1 report packet into normal multicast
+// egress rule as well.
+func (f *featureMulticast) igmpEgressFlow() []binding.Flow {
```

Comment:
I don't think you should mention NP counting issue. If NP matches traffic wrongly in the flow, the stats is cannot be correct.